src/daily-quest/daily-quest.service.ts

```diff
@@ -174,9 +206,18 @@ export default class DailyQuestService {
       },
       orderBy: { [sortBy]: sortOrder },
       take: limit,
+      include: {
+        daily_quest_completions: {
+          where: { date: date },
+        },
+      },
     });
 
-    return dailyQuests.map(toDailyQuestModel);
+    return dailyQuests.map((dailyQuests) => ({
```

Comment:
please, do not name array and its member the same name. it's confusing. 
```suggestion
    return dailyQuests.map((quest) => ({
```

Reply:
Thanks, fixed!